core_screenshot_config.yaml

```diff
@@ -78,11 +78,12 @@ primary:
     message: waiting 30 sec to load TX
 
   UT:
+  overseerScript: page.manualWait(); await page.waitForDelay(30000); page.done();
```

Comment:
I think this might not work since it needs another indent - were you by any chance able to test this locally?

Reply:
I was not, but I defer to your knowledge on this. Should I submit another pull request with the indent?